test/src/main/scala/scala/test/twitter_scrooge/grep.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+if grep -q $1 $2 ; then
+  echo "ERROR: found $1"
```

Comment:
Maybe add what was expected? Either in the assertion or in the test name since 6 months from now I won’t remember :)